src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/ApprovalController.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.companieshouse.web.accounts.controller.smallfull;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping(value = "/company/{companyNumber}/small-full/approval")
```

Comment:
This will need to follow the same format as the `BalanceSheetController` i.e.: `/company/{companyNumber}/transactions/{transactionId}/company-accounts/{companyAccountsId}/small-full/approval`. That way you'll have access to the `transactionId` as a `PathVariable` to close the transaction in the POST handler method.